### PR TITLE
Py36 fix

### PR DESCRIPTION
**Pull request by esvhd**
Hi,

This is related to #3 changes to make these scripts work for Python 3.x.

I tested these changes with `Anaconda Python 3.6`. 

Hope this helps.

Thanks.

### CODE REVIEW

File: data_util.py [hunk 1]
```diff
@@ -2,10 +2,15 @@
 import os
 from os import path
 import urllib
+import urllib.request as request
```
Comment (vinhkhuc) [inline data_util.py]: Python 2 doesn't have urllib.request. This import will make the code not able to run in Python 2.

**Review by vinhkhuc — CHANGES_REQUESTED** [review]
Thanks for the PR. Looks good to me overall. Just found one small issue with it.

**Comment by esvhd** [comment]
Good spot, let me fix this later. Thx

**Comment by esvhd** [comment]
fixed, import now conditional on major python version number.

**Comment by vinhkhuc** [comment]
Great! Thank you very much.

**Review by vinhkhuc — APPROVED** [review]

**Comment by esvhd** [comment]
Thanks!!

**Pull request merged**